<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_PreflightVerifier</class>
 <widget class="QWidget" name="Prefs_PreflightVerifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Preflight Verifier</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showPagesWithoutErrorsCheckBox">
     <property name="text">
      <string>Show pages without errors</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="showNonPrintingLayerErrorsCheckBox">
     <property name="text">
      <string>Show errors for items on layers that will not print</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Minimum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Profile Setup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>597</width>
        <height>545</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Output Profile:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="currentProfileComboBox">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="addProfilePushButton">
           <property name="text">
            <string>Add New Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeProfilePushButton">
           <property name="text">
            <string>Remove Profile</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Profile Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ignoreAllErrorsCheckBox">
         <property name="text">
          <string>Ignore all errors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="autoCheckBeforePrintExportCheckBox">
         <property name="text">
          <string>Automatic check before printing or exporting</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkMissingGlyphsCheckBox">
         <property name="text">
          <string>Check for missing glyphs</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkItemsNotOnAPageCheckBox">
         <property name="text">
          <string>Check for items not on a page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTextOverflowCheckBox">
         <property name="text">
          <string>Check for overflow in text frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkTransparenciesCheckBox">
         <property name="text">
          <string>Check for used transparencies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkMissingImagesCheckBox">
         <property name="text">
          <string>Check for missing images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkPartFilledImageFramesCheckBox">
         <property name="text">
          <string>Check for part filed image frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkImageResolutionCheckBox">
         <property name="text">
          <string>Check image resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="formAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Minimum Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="minimumResolutionSpinBox">
           <property name="suffix">
            <string> dpi</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>4800</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="maximumResolutionSpinBox">
           <property name="suffix">
            <string> dpi</string>
           </property>
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>4800</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Maximum Resolution:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkPlacedPDFCheckBox">
         <property name="text">
          <string>Check for placed PDF files</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkGIFsCheckBox">
         <property name="text">
          <string>Check for GIF images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkPDFAnnotFieldsCheckBox">
         <property name="text">
          <string>Check for PDF Annotations and Fields</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkVisiblePrintableLayersCheckBox">
         <property name="text">
          <string>Check for Visible/Printable mismatch in layers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkNonPrintableLayersCheckBox">
         <property name="text">
          <string>Ignore non-printable layers</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>124</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
